src/raiden_libs/contract_info.py

```diff
@@ -44,8 +44,9 @@ def get_contract_addresses_and_start_block(
 
     # Get deployed addresses for those contracts which have no overwrites
     addresses = {
-        c: address_overwrites.get(
-            c, to_canonical_address(contract_data["contracts"][c]["address"])
+        c: (
+            address_overwrites.get(c)
+            or to_canonical_address(contract_data["contracts"][c]["address"])
```

Comment:
Is there a difference here?

Reply:
Yes, a very subtle one. In the old version the fallback was evaluated in every case, causing a key error for the deposit contract.